x-pack/plugins/observability_solution/inventory/server/routes/has_data/get_has_data.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ENTITY_LATEST, entitiesAliasPattern } from '@kbn/entities-schema';
+import type { Logger } from '@kbn/core/server';
+
+import { esqlResultToPlainObjects } from '@kbn/observability-utils/es/utils/esql_result_to_plain_objects';
+import { type ObservabilityElasticsearchClient } from '@kbn/observability-utils/es/client/create_observability_es_client';
+import {
+  getEntityDefinitionIdWhereClause,
+  getEntityTypesWhereClause,
+} from '../entities/query_helper';
+
+const ENTITIES_LATEST_ALIAS = entitiesAliasPattern({
+  type: '*',
+  dataset: ENTITY_LATEST,
+});
```

Comment:
Can you import it from https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/inventory/common/entities.ts#L21

Reply:
updated it here https://github.com/elastic/kibana/pull/193755/commits/21bf56497bbc8b929c6371558cb7339b608e8fca